test/unrunnableTestCases.csv

```diff
@@ -1304,6 +1251,7 @@ xs-error-047,AssertionError: expected [Function] to throw error including 'FOER0
 xs-error-048,AssertionError: expected [Function] to throw error including 'XPDY0050' but got 'XPST0003: Unable to parse XPath: 1 treat as xs:error. SyntaxError: Expected " ", "!", "!=", "(:", "*", "+", ",", "-", "/", "//", "<", "<<", "<=", "=", "=>", ">", ">=", ">>", "\\n", "\\r", "\\t", "and", "cast", "castable", "div", "eq", "except", "ge", "gt", "idiv", "instance", "intersect", "is", "le", "lt", "mod", "ne", "or", "to", "union", "|", "||", or end of input but "t" found.'
 xs-error-049,Error: XPST0003: Unable to parse XPath: (() treat as xs:error?) => empty(). SyntaxError: Expected " ", "!", "!=", "(:", ")", "*", "+", ",", "-", "/", "//", "<", "<<", "<=", "=", "=>", ">", ">=", ">>", "\n", "\r", "\t", "and", "cast", "castable", "div", "eq", "except", "ge", "gt", "idiv", "instance", "intersect", "is", "le", "lt", "mod", "ne", "or", "to", "union", "|", or "||" but "t" found.
 xs-float-004,AssertionError: expected [Function] to throw an error
+K-DayTimeDurationAdd-3,AssertionError: Expected XPath xs:dayTimeDuration("P3DT4H3M3.100S") + xs:dayTimeDuration("P3DT12H31M56.303S") eq xs:dayTimeDuration("P6DT16H34M59.403S") to resolve to true: expected false to be true
```

Comment:
Yep, comes down to <some_big_value_in_seconds>.403 % 60 = <number_of_seconds_to_display>.4029999975.....